src/bodywork/cli/cli.py

```diff
@@ -394,7 +392,6 @@ def cronjob(args: Namespace) -> None:
     """Cronjob command handler.
```

Comment:
Thanks for cleaning these up.

Reply:
No problem, I naturally clean up as I come across things.